cookbook-react/src/recipes/HOCs/Spinner/components/Loading/__snapshots__/index.test.tsx.snap

```diff
@@ -1,13 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This wasn't being used?

Reply:
no, it was not, because I moved the Loading to a separate recipe